### PR TITLE
fixes issue #984: Build fails from 5.2.4 beta tarball due to missing version.txt

### PR DESCRIPTION
**Pull request by tst-eclamar**
fixes issue #984 

This is a very simple change.  However, to test it requires some temporary changes to build_release_archive.sh.  Normally, we'd checkout 'google/earthenterprise', but to test the code we need to checkout the branch that the modified code currently lives in.  The lines marked "# Temporary" will either be changed or deleted.

### CODE REVIEW

File: scripts/build_release_archive.sh [hunk 1]
```diff
@@ -37,7 +37,16 @@ CLONE_DIR=`mktemp -d`
...
 # Download the repository
 cd "${CLONE_DIR}"
-git clone -b "${1}" --depth 1 "https://github.com/google/${REPO_NAME}.git" || exit
+git clone -b "${1}" "https://github.com/tst-eclamar/${REPO_NAME}.git" || exit # TEMPORARY: tst-eclamar => google
+
+cd "${CLONE_DIR}"/earthenterprise/earth_enterprise/src
+git remote add upstream git://github.com/google/earthenterprise.git # TEMPORARY
+git fetch upstream # TEMPORARY
+git pull upstream master # TEMPORARY
+scons version_files
```
Comment (tst-rwildes) [inline scripts/build_release_archive.sh]: Probably add a comment like "generate version files based on tags within git"

File: scripts/build_release_archive.sh [hunk 2]
```diff
@@ -37,7 +37,16 @@ CLONE_DIR=`mktemp -d`
...
+cd ../..
+git clean -f -d -x -e version.txt
```
Comment (tst-rwildes) [inline scripts/build_release_archive.sh]: Also comment: "remove all un-tracked files except for version.txt."

**Comment by tst-ccamp** [comment]
@tst-eclamar Please update the issue number in the title and description

**Comment by tst-rwildes** [comment]
@tst-eclamar  I think this PR is based off of master.  We should probably re-do it and base if off of release_5.2.4 branch and target that branch for the PR.  Also remember to include the github issue number in the title and description.

**Comment by tst-eclamar** [comment]
Due to my failings and other issues, this is now a corrupt PR and will be closed.

File: scripts/build_release_archive.sh [hunk 3]
```diff
@@ -37,7 +37,18 @@ CLONE_DIR=`mktemp -d`
 
 # Download the repository
 cd "${CLONE_DIR}"
-git clone -b "${1}" --depth 1 "https://github.com/google/${REPO_NAME}.git" || exit
+git clone -b "${1}" "https://github.com/tst-eclamar/${REPO_NAME}.git" || exit # TEMPORARY: tst-eclamar => google
+
+cd "${CLONE_DIR}"/earthenterprise/earth_enterprise/src
+git remote add upstream git://github.com/google/earthenterprise.git # TEMPORARY
+git fetch upstream # TEMPORARY
+git pull upstream master # TEMPORARY
```
Comment (tst-mbiancalana) [inline scripts/build_release_archive.sh]: When are these lines marked TEMPORARY be removed or changed?

**Pull request closed without merging**